docs/infrastructure/koji-user-management.md

```diff
@@ -0,0 +1,168 @@
+which contains the full cert chains for these CAs.
+
+!!! note
+    The file also contains the cert chains for the following two CAs, which are used for build hosts and automated users.
+
+    - koji.chtc.wisc.edu
+    - InCommon RSA Server CA
+
+You can usually guess which CA signed the user's cert from their DN.
+If unsure, ask them for the output of:
+``` console
+$ openssl x509 -in <CERTFILE> -noout -issuer
+```
+If the CA is not one of the above ones, you will have to add the CA.
+Get permission to do this from the Software Manager, then follow the procedure on the [koji-hub setup page](https://opensciencegrid.github.io/technology/infrastructure/koji-hub-setup/#adding-cas-for-user-authentication).
+
+
+The user's Koji username will be the _first_ Common Name (CN) of their certificate subject.
+For example:
+``` console
+$ openssl x509 -in <CERTFILE> -noout -subject
+subject= /DC=org/DC=cilogon/C=US/O=Fermi National Accelerator Laboratory/OU=People/CN=Matyas Selmeci/CN=UID:matyas
+```
+will result in the Koji username "Matyas Selmeci".
+To add the user, run
+``` console
+$ osg-koji add-user "<USERNAME>"
+```
+This will create the user but will give them no permissions.
+
+!!! warning
+    It is effectively impossible to delete a user.
+    If you get the name wrong, follow the [instructions below for renaming a user](#handling-user-cert-changes-renaming-a-user) below.
+
+To give them permissions, use the command
+``` console
+$ osg-koji grant-permission <PERMISSION> "<USERNAME>"
+```
+Generally, the permissions you should give are (more explanation below):
+
+- most people should be given `build` and `repo`
+- HCC people should also be given `hcc-build`
+- S&R people should also be given `software-team`
+- Security people should also be given `security-team`
+- Ops people should also be given `operations-team`
+- other software devs should also be given `software-contributor`
+- few people should be given `admin`
+
+To revoke permissions, use the command
+``` console
+$ osg-koji revoke-permission <PERMISSION> "<USERNAME>"
+```
+
+### Permissions details
+
+To list the available permissions, run
+``` console
+$ osg-koji list-permissions
+```
+
+The important permissions are:
+
+- **repo**: the ability to run `osg-koji regen-repo`.
+  Should be given to anyone that builds or tests packages.
+- **build**: the ability to build (but not tag!) any package.
+  Should be given to anyone that builds packages, including automated users.
+  This perm is necessary but _not_ sufficient for them to build into the development repos;
+  it _is_ enough to let them do scratch builds.
+- **software-contributor**: the ability to build into _any_ of the development repos _and_ promote into the osg-testing repos (but not any other repos).
+  Should be given to people that build specific software but do not belong to the OSG S&R team or HCC.
+- **operations-team**: the ability to build into development and promote into testing the `vo-client` package.
```

Comment:
We build the `vo-client` again, do we need this any more?

Reply:
Nope. Also the operations team is going away.
Also, "software-team" and "release-team" can be combined now.

I'd like to make the Koji changes before updating the document.

Reply:
ok, that's fine with me